tests/agents/test_details_agent.py

```
@@ -382,13 +383,13 @@ def test_run(self, mock_fix_ref, mock_validation_invoke):
 
         api_response = ComponentApiSurfaces(api_surfaces=[])
         relation_response = ComponentRelations(components_relations=[])
-        mock_validation_invoke.side_effect = [cluster_response, final_response, api_response, relation_response]
+        mock_invoke_repair_validate.side_effect = [cluster_response, final_response, api_response, relation_response]
         mock_fix_ref.return_value = final_response
 
         analysis, subgraph_results = agent.run(self.test_component)
```

Ruff (0.15.20)

[warning] Unpacked variable `subgraph_results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)